java-dynamic-sqs-listener-core/src/main/java/com/jashmore/sqs/util/ResizableSemaphore.java

```diff
@@ -0,0 +1,35 @@
+package com.jashmore.sqs.util;
+
+import lombok.Getter;
+
+import java.util.concurrent.Semaphore;
+
+/**
+ * Semaphore that is able to dynamically update the number of available permits.
+ */
+@Getter
+public class ResizableSemaphore extends Semaphore {
+    private static final long serialVersionUID = 1L;
```

Comment:
You should pull these out if the only reason they are in here is to appease the warning for checkstyle/findbugs

Reply:
Yeah maven-compiler-plugin warnings. I tried to find ways to not have to do it online and everyone was like you should do it.

I can try and figure it out again